spec/spec_helper.rb

```diff
@@ -11,8 +11,14 @@
 
 require 'rspec'
 require 'webmock/rspec'
-require "naver/searchad/api"
-require "naver/searchad/api/core/base_service"
+require 'naver/searchad/api'
+require 'naver/searchad/api/core/base_service'
+require 'naver/searchad/api/auth'
+require 'naver/searchad/api/campaign/service'
+
+ENV['NAVER_API_KEY'] = '0100000000f2e75122770874cb904034e7e27f5815c21af53a93f25b0f05f0ce97f263650c'
+ENV['NAVER_API_SECRET'] = 'AQAAAADy51Eidwh0y5BANOfif1gVwHjWG4MrXg6Mbh54YHY4MQ=='
+ENV['NAVER_API_CLIENT_ID'] = '1077530'
 
```

Comment:
this should be testing credentials, I just put them to run to collect data for mockup, should be removed later